packages/types/src/identity-types.ts

```diff
@@ -4,9 +4,21 @@ export interface IIdentity {
   type: TYPE;
   // the identification itself
   value: string;
+  // extra information
+  extra?: IExtraInfoIdentity;
```

Comment:
alternative proposition:
- Leave IIdentity untouched
- Create a ISmartContractIdentity:
```typescript
interface ISmartContractIdentity extends IIdentity {
   network:string;
}
```

usage: 
```typescript
if(type === TYPE.ETHEREUM_SMART_CONTRACT){
  const network =(identity as ISmartContractIdentity).network;
}
```

Reply:
This looks good to me. We only have to make sure that doing this the network will be correctly added to the transaction data. I guess so, since in runtime we don't know about types anymore, but worth having a look (or even better, writing a test).